gui/qt/paytoedit.py

```diff
@@ -177,9 +177,6 @@ def get_outputs(self, is_max):
     def lines(self):
         return self.toPlainText().split('\n')
 
-    def is_multiline(self):
```

Comment:
this method is used in this class, around line 206 :P

Reply:
I saw that. My thought is that it does extend CompletionTextEdit, so that functionality should still be available. Unless I need to use ```parent.is_multiline()``` instead of ```self.is_multiline()``` on line 206

Reply:
It should be fine; sorry, I was not paying enough attention. See EDIT above.